public/assets/js/members.js

```diff
@@ -36,7 +36,7 @@ $(document).ready(function() {
       // $(".group-names").append(newList);
     });
   });
-});
```

Comment:
It seemed as though having the }); here made it so the rest of the code wasn't included.  Please double check this one closely in case I was mistaken.